modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/QueryDescriptor.java

```diff
@@ -171,6 +187,9 @@ public String queryInitiatorId() {
         if (schemaName != null ? !schemaName.equals(that.schemaName) : that.schemaName != null)
             return false;
 
+        if (!Objects.equals(label, that.label))
```

Comment:
Why should the label be taken into account here? It looks like we need to use parameters that can affect execution, but label it as just a marker